.changeset/calm-books-connect.md

```
@@ -0,0 +1,5 @@
+---
+'mastracode': minor
+---
+
+Added `/connect` for provider accounts and API keys. `/login` remains available as an alias.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)